.eslintrc

```diff
@@ -3,3 +3,6 @@ parser: babel-eslint
 
 env:
   jest: true
+
+globals:
+  ReactClass: true
```

Comment:
eslint consider flow types and it works with rules like 'no-undef`, so we need to add it as global.

but maybe there is better solution with some plugin etc., I'll take a look